### ISSUE TITLE
Register for an account in the same interview

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-EFSPIntegration#241]
Similar to how we did the Payments stuff in the same interview, we should consider letting people register for a new account in the same interview.

The different interview approach is easy from a development standpoint, but it disorients users, who aren't sure how to get back to the main interview and are now in the wrong tab.

**Comment by nonprofittechy** [comment]
Agreed! I think I'm responsible for this original choice and it was only meant as a placeholder to get through the certification.

**Comment by BryceStevenWilley** [comment]
This is now a part of this package, since #308
